charts/kube-prometheus-stack/templates/prometheus/rules-1.14/general.rules.yaml

```diff
@@ -29,7 +29,10 @@ spec:
         description: '{{`{{`}} printf "%.4g" $value {{`}}`}}% of the {{`{{`}} $labels.job {{`}}`}}/{{`{{`}} $labels.service {{`}}`}} targets in {{`{{`}} $labels.namespace {{`}}`}} namespace are down.'
         runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-targetdown
         summary: One or more targets are unreachable.
-      expr: 100 * (count(up == 0) BY (job, namespace, service) / count(up) BY (job, namespace, service)) > 10
+      expr: |-
+        {{ .Values.defaultRules.expressionPrefix }}
```

Comment:
This is a change in a generated file (see the comment at the top of the file). When the next rules-sync is executed, your changes will be reverted.